inc/functions/options.php

```diff
@@ -412,6 +412,7 @@ function get_rocket_exclude_defer_js() {
 		'www.uplaunch.com',
 		'google.com/recaptcha',
 		'widget.reviews.co.uk',
+		'jupiterx/lib/admin/assets/lib/webfont/webfont.min.js'
```

Comment:
The risk here is that someone might have renamed the folder name for the theme, and so the exclusion doesn't work.

The quick solution is to remove the folder name from the path.

Reply:
Basically to keep only: `lib/admin/assets/lib/webfont/webfont.min.js`

And Unit tests are failing . Should I correct those?